docs/Developers/Newcomers_Guide.rst

```diff
@@ -140,6 +140,12 @@ to be part of the Newcomer team. The invitation will be sent by mail and you
 will have to accept it to join. If you don't find the invitation, accept it
 `here <https://github.com/coala>`__.
 
+.. note::
```

Comment:
I would add an empty line here to conform to the style of other notes in this document